packages/orchestration/src/utils/registry.js

```diff
@@ -121,9 +127,10 @@ export const convertChainInfo = async registry => {
       ibcData
         .map(datum => toConnectionEntry(datum, name, chainInfo))
         // sort alphabetically for consistency
-        .sort(([a], [b]) => (a && b ? a.localeCompare(b) : 0)),
```

Comment:
why this change?

Reply:
I'm seeing unsorted output in some cases.

I'm not sure this change fixes it, but it reminds me to think more about it.